Controllers/BookingController.cs

```diff
@@ -28,4 +28,11 @@ public async Task<IActionResult> GetBookings(string userId)
         var bookings = await _bookingService.GetMyBookings(Guid.Parse(userId));
         return Ok(bookings);
     }
+
+    [HttpPut("completeBooking/{bookingId}")]
+    public async Task<IActionResult> SetBookingAsCompleted(string bookingId)
+    {
+        var booking = await _bookingService.SetBookingAsCompleted(bookingId);
```

Comment:
Perhaps this needs error handling, if something is wrong with booking.
example:
```
 try
    {
        var booking = await _bookingService.SetBookingAsCompleted(bookingId);

        if (booking == null)
        {
            return NotFound($"Booking with ID {bookingId} not found.");
        }

        return Ok();
    }
    catch (Exception ex)
    {
        return StatusCode(500, "Some sort of error occurred.");
    }
```